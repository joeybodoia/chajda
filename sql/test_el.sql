CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."');
