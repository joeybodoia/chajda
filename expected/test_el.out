CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                    spacy_lemmatize                                                    
-----------------------------------------------------------------------------------------------------------------------
   παραδειγματική:5 δοκιμαστική:6 πρόταση:7 βάζω:9 μετάφραση:11 google:12 δημιουργία:15 δοκιμαστικών:16 περιπτώσεων:17
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                        spacy_lemmatize                                         
------------------------------------------------------------------------------------------------
   παραδειγματική δοκιμαστική πρόταση βάζω μετάφραση google δημιουργία δοκιμαστικών περιπτώσεων
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                          spacy_lemmatize                                                                           
--------------------------------------------------------------------------------------------------------------------------------------------------------------------
   αυτή:2 είναι:3 η:4 παραδειγματική:5 δοκιμαστική:6 πρόταση:7 που:8 βάζω:9 στη:10 μετάφραση:11 google:12 για:13 τη:14 δημιουργία:15 δοκιμαστικών:16 περιπτώσεων:17
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                                      spacy_lemmatize                                                       
----------------------------------------------------------------------------------------------------------------------------
   αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη μετάφραση google για τη δημιουργία δοκιμαστικών περιπτώσεων
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                           spacy_lemmatize                                                            
--------------------------------------------------------------------------------------------------------------------------------------
   ":2 παραδειγματική:6 δοκιμαστική:7 πρόταση:8 βάζω:10 μετάφραση:12 google:13 δημιουργία:16 δοκιμαστικών:17 περιπτώσεων:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                           spacy_lemmatize                                            
------------------------------------------------------------------------------------------------------
   " παραδειγματική δοκιμαστική πρόταση βάζω μετάφραση google δημιουργία δοκιμαστικών περιπτώσεων . "
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                                  spacy_lemmatize                                                                                  
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 αυτή:3 είναι:4 η:5 παραδειγματική:6 δοκιμαστική:7 πρόταση:8 που:9 βάζω:10 στη:11 μετάφραση:12 google:13 για:14 τη:15 δημιουργία:16 δοκιμαστικών:17 περιπτώσεων:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                                         spacy_lemmatize                                                          
----------------------------------------------------------------------------------------------------------------------------------
   " αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη μετάφραση google για τη δημιουργία δοκιμαστικών περιπτώσεων . "
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                    spacy_lemmatize                                                    
-----------------------------------------------------------------------------------------------------------------------
   παραδειγματική:5 δοκιμαστική:6 πρόταση:7 βάζω:9 Μετάφραση:11 Google:12 δημιουργία:15 δοκιμαστικών:16 περιπτώσεων:17
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                        spacy_lemmatize                                         
------------------------------------------------------------------------------------------------
   παραδειγματική δοκιμαστική πρόταση βάζω Μετάφραση Google δημιουργία δοκιμαστικών περιπτώσεων
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                          spacy_lemmatize                                                                           
--------------------------------------------------------------------------------------------------------------------------------------------------------------------
   Αυτή:2 είναι:3 η:4 παραδειγματική:5 δοκιμαστική:6 πρόταση:7 που:8 βάζω:9 στη:10 Μετάφραση:11 Google:12 για:13 τη:14 δημιουργία:15 δοκιμαστικών:16 περιπτώσεων:17
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                                      spacy_lemmatize                                                       
----------------------------------------------------------------------------------------------------------------------------
   Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                           spacy_lemmatize                                                            
--------------------------------------------------------------------------------------------------------------------------------------
   ":2 παραδειγματική:6 δοκιμαστική:7 πρόταση:8 βάζω:10 Μετάφραση:12 Google:13 δημιουργία:16 δοκιμαστικών:17 περιπτώσεων:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                           spacy_lemmatize                                            
------------------------------------------------------------------------------------------------------
   " παραδειγματική δοκιμαστική πρόταση βάζω Μετάφραση Google δημιουργία δοκιμαστικών περιπτώσεων . "
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                                  spacy_lemmatize                                                                                  
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 Αυτή:3 είναι:4 η:5 παραδειγματική:6 δοκιμαστική:7 πρόταση:8 που:9 βάζω:10 στη:11 Μετάφραση:12 Google:13 για:14 τη:15 δημιουργία:16 δοκιμαστικών:17 περιπτώσεων:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                                         spacy_lemmatize                                                          
----------------------------------------------------------------------------------------------------------------------------------
   " Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων . "
(1 row)

SELECT spacy_tsvector('el',' "Αυτή είναι η παραδειγματική δοκιμαστική πρόταση που βάζω στη Μετάφραση Google για τη δημιουργία δοκιμαστικών περιπτώσεων."');
                                                            spacy_tsvector                                                             
---------------------------------------------------------------------------------------------------------------------------------------
 'google':12 'βάζω':9 'δημιουργία':15 'δοκιμαστική':6 'δοκιμαστικών':16 'μετάφραση':11 'παραδειγματική':5 'περιπτώσεων':17 'πρόταση':7
(1 row)

